modules/release-notes/snip-removed-functionality-rhidp-2794.adoc

```diff
@@ -0,0 +1,34 @@
+[id="removed-functionality-rhidp-2794"]
+= Breaking changes in Backstage release 1.29
```

Comment:
if you flatten this whole section down to a list of links, you could move it under the item about moving to 1.29.2.

```
* link:https://github.com/backstage/backstage/releases/tag/v1.27.0[Backstage 1.27 release notes] 
* link:https://github.com/backstage/backstage/blob/v1.27.0/docs/releases/v1.27.0-changelog.md[Backstage 1.27 changelog]
* link:https://github.com/backstage/backstage/releases/tag/v1.28.0[Backstage 1.28 release notes] 
* link:https://github.com/backstage/backstage/blob/v1.28.0/docs/releases/v1.28.0-changelog.md[Backstage 1.28 changelog]
* link:https://github.com/backstage/backstage/releases/tag/v1.29.0[Backstage 1.29 release notes] 
* link:https://github.com/backstage/backstage/blob/v1.29.2/docs/releases/v1.29.0-changelog.md[Backstage 1.29 changelog]


.Additional links:
* link:https://issues.redhat.com/browse/RHIDP-2794[RHIDP-2794]
* link:https://issues.redhat.com/browse/RHIDP-2847[RHIDP-2847]
* link:https://issues.redhat.com/browse/RHIDP-2796[RHIDP-2796]
```

Reply:
That's nice